selectors/emojis.ts

```diff
@@ -23,23 +22,26 @@ export const getEmojiMap = createSelector(
     },
 );
 
-export const getShortcutReactToLastPostEmittedFrom = (state: GlobalState) => state.views.emoji.shortcutReactToLastPostEmittedFrom;
-
-export const getRecentEmojis = createSelector(
-    'getRecentEmojis',
-    (state: GlobalState) => LocalStorageStore.getRecentEmojis(getCurrentUserId(state)),
-    (recentEmojis) => {
-        if (!recentEmojis) {
-            return [];
-        }
+export const getShortcutReactToLastPostEmittedFrom = (state: GlobalState) =>
+    state.views.emoji.shortcutReactToLastPostEmittedFrom;
 
-        const recentEmojisArray: string[] = JSON.parse(recentEmojis);
-        return recentEmojisArray;
-    },
-);
+export function getRecentEmojis(state: GlobalState): RecentEmojiData[] {
```

Comment:
This needs to go back to being wrapped in `createSelector` so that its results are memoized. Otherwise, we're returning a new array from JSON.parse each time this is called which will cause various any React components relying on this to re-render constantly